Clean up and simplify the run of show content (#866)
writethedocs/www: docs/conf/portland/2019/sponsors/prospectus.rst

- sponsorship@writethedocs.org

.. _ticket: https://ti.to/writethedocs/write-the-docs-{{ shortcode }}-{{ year }}/
.. _tickets: https://ti.to/writethedocs/write-the-docs-{{ shortcode }}-{{ year }}/

Run of Show
===========

> This Run of Show hopefully will give you more context about the event and answer some of the questions you may have. 

Reviewer: ```suggestion
This Run of Show provides more context about the event and answers some common questions you may have. 
```

> Please give us feedback if there is anything missing or information that would be useful for you.

Reviewer: ```suggestion
Please let us know if there is any information missing that would be useful for you.
```

Sponsorship schedule
--------------------

> * **SUNDAY**: The swag tables are available during the Writing Day, if you want to put out swag items early. Sponsor booths are available for setup on Sunday from 10am-4pm, but don't need people staffing them yet. You're also welcome to lead a Writing Day project on your companies documentation, if it is open source.

Reviewer: ```suggestion
* **SUNDAY**: The swag tables are available during the Writing Day, if you want to put out swag items early. Sponsor booths are available for setup on Sunday from 10am-4pm, but don't need people staffing them. You're also welcome to lead a Writing Day project if your documentation is open source.
```
